src/policy/immortalspace.rs

```diff
@@ -45,6 +45,15 @@ impl<VM: VMBinding> SFT for ImmortalSpace<VM> {
         );
         old_value == self.mark_state
     }
+    fn pin_object(&self, _object: ObjectReference) -> bool {
+        false
+    }
+    fn unpin_object(&self, _object: ObjectReference) -> bool {
+        false
+    }
+    fn is_object_pinned(&self, _object: ObjectReference) -> bool {
+        false
```

Comment:
Should this return `true`? since all objects in this space are non-movable.

Reply:
I guess it could. It's a tricky question, the object is not pinned in the sense that the pinning bit is not set, but since the object is in a non-movable space, it is technically pinned. 
That being said It would probably not cause any harm returning true here and in the code below.